examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -310,15 +338,6 @@
     "\n",
     "*Head back to the notebook from part 1. When you're done watching the accuracy of your application, stop the cell (hit the little \"stop\" square in the notebook toolbar).*"
    ]
-  },
```

Comment:
As discussed, put the Cleanup section here. Find a fun warning/caution image to put at the start of the cleanup section and make sure it's clear to run cleanup only when you are completely done with the tutorial.